examples/nft-quest/nuxt.config.ts

```diff
@@ -19,11 +19,7 @@ export default defineNuxtConfig({
   $production: {
     runtimeConfig: {
       public: {
-        chain: zksyncSepoliaTestnet,
-        contracts: {
-          nft: "0x4D533d3B20b50b57268f189F93bFaf8B39c36AB6",
-          paymaster: "0x60eef092977DF2738480a6986e2aCD10236b1FA7",
-        },
+        defaultChainId: zksyncSepoliaTestnet.id,
```

Comment:
Let's just do same as in auth server, use default chain env by default or testnet if the env is missing. I thought that was the idea of this PR, but env isn't used

Reply:
Runtimeconfig vars will automatically get replaced if it follows the naming scheme: "NUXT_PUBLIC_{public runtime config prop}". Writing it in is redundant and ignored if the env var is defined.